doc/classes/FileAccess.xml

```diff
@@ -312,6 +336,33 @@
 				[b]Note:[/b] This is an offset, so you should use negative numbers or the cursor will be at the end of the file.
 			</description>
 		</method>
+		<method name="set_hidden_attribute" qualifiers="static">
+			<return type="int" enum="Error" />
+			<param index="0" name="file" type="String" />
+			<param index="1" name="hidden" type="bool" />
+			<description>
+				Sets file [code]hidden[/code] attribute.
+				[b]Note:[/b] This method is implemented on iOS, macOS, and Windows.
+			</description>
+		</method>
+		<method name="set_read_only_attribute" qualifiers="static">
+			<return type="int" enum="Error" />
+			<param index="0" name="file" type="String" />
+			<param index="1" name="ro" type="bool" />
+			<description>
+				Sets file [code]read only[/code] attribute.
+				[b]Note:[/b] This method is implemented on iOS, macOS, and Windows.
+			</description>
+		</method>
+		<method name="set_unix_permissions" qualifiers="static">
+			<return type="int" enum="Error" />
+			<param index="0" name="file" type="String" />
+			<param index="1" name="permissions" type="int" />
+			<description>
+				Sets file UNIX permissions.
```

Comment:
`chmod 755` is octal, so it takes`0x1ED` (GDScript do not have octal support). Added a list of values to combine.

Reply:
Can we define an enum for these values then? The octal format is widespread and so it's easy to read with some experience, and this is less so. But maybe it's an overkill.

Reply:
An enum would be good for usability, but I think this might be a niche enough feature that it may not be worth it to add more cognitive overload in the FileAccess API by adding an enum with all these values.

I wouldn't oppose it if @bruvzg thinks it's worth adding, but we can also just wait to see if there's demand.

Reply:
Changed it to bitfield enum.